### PR TITLE
Fixing and Re-recording CLU live tests

### PR DESCRIPTION
**Pull request by ZulaMostafa**

### CODE REVIEW

File: sdk/cognitivelanguage/test-resources.json [hunk 1]
```diff
@@ -23,42 +23,42 @@
...
+        // },
+        // "AZURE_CONVERSATIONS_WORKFLOW_PROJECT_NAME": {
+        //     "type": "string",
+        //     "value": "antischTwo"
+        // },
+        // "AZURE_CONVERSATIONS_WORKFLOW_DEPLOYMENT_NAME": {
+        //     "type": "string",
+        //     "value": "production"
+        // },
+        // "QUESTIONANSWERING_ENDPOINT": {
```
Comment (heaths) [inline sdk/cognitivelanguage/test-resources.json:51]: Even though you're commenting this out, use the correct "AZURE_" prefix. Make sure the rest line up. That way, if/when we do restore these, they are already correct and there's no confusion.

**Review by heaths — APPROVED** [review]
Before I merge, were these recorded with the local env vars that @mshaban-msft passed along internally? I updated our shared Key Vault this afternoon using those values except for `qna-key` and `qna-uri` but, as we're discussing in internally, maybe we should - expecting that QA live tests would then fail (but not important right now).

File: sdk/cognitivelanguage/test-resources.json (hunk 1 above)
Comment (ZulaMostafa) [reply sdk/cognitivelanguage/test-resources.json:51]: Yes, they were recorded with the vars that Mohammed passed.

**Review by heaths — APPROVED** [review]

**Comment by heaths** [comment]
Be sure eventually to update _QuestionAnsweringTestEnvironment.cs_ with the new env vars. Technically, these should still work since the base `TestEnvironment` class falls back to prepending "AZURE_", but while we're busy fixing up QA anyway, let's get everything in sync for easier troubleshooting, should the need ever arise.

**Pull request merged**